public/src/components/channelManagement/campaigns/CampaignsList.tsx

```diff
@@ -56,21 +58,36 @@ const CampaignsList = ({
     return 'outlined';
   };
 
-  // When we get creation timestamps, we are thinking of allowing users to sort the list either alphabetically or by most recently created. For now, only the alphabetically sorted version is possible
-  const sortedCampaigns = [...campaigns];
+  const filterCampaigns = (campaignArray: Campaigns) => {
+    return campaignArray.filter(c => {
+      if (!campaignSearch) {
```

Comment:
not terribly important but if you move this `if (!campaignSearch)` check outside of the filter then the browser will not have to do pointless work every time `campaignSearch` is empty

Reply:
It's a good idea!